### PR TITLE
Add kubedistro images

### PR DESCRIPTION
**Pull request by mdsgabriel**
*Issue #, if available:*

*Description of changes:*
Add KubeDistro images to the import-images command

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/380#" title="Author self-approved">mdsgabriel</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere/blob/main/OWNERS)~~ [mdsgabriel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by g-gaston — APPROVED** [review]

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/380#pullrequestreview-770867216" title="Approved">g-gaston</a>*, *<a href="https://github.com/aws/eks-anywhere/pull/380#" title="Author self-approved">mdsgabriel</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere/blob/main/OWNERS)~~ [g-gaston,mdsgabriel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cmd/eksctl-anywhere/cmd/importimages.go
Comment (abhay-krishna) [inline cmd/eksctl-anywhere/cmd/importimages.go:68]:
```suggestion
			return fmt.Errorf("error importing image %s: %v", image.URI, err)
```

File: cmd/eksctl-anywhere/cmd/importimages.go
Comment (abhay-krishna) [inline cmd/eksctl-anywhere/cmd/importimages.go:62]:
```suggestion
			return fmt.Errorf("error importing image %s: %v", image.URI, err)
```

File: cmd/eksctl-anywhere/cmd/importimages.go
Comment (mdsgabriel) [reply cmd/eksctl-anywhere/cmd/importimages.go]: fixed

File: cmd/eksctl-anywhere/cmd/importimages.go
Comment (mdsgabriel) [reply cmd/eksctl-anywhere/cmd/importimages.go]: thanks! fixed

**Pull request merged**
